app/components/course-page/course-stage-step/second-stage-instructions-card.hbs

```diff
@@ -1,4 +1,10 @@
-<CoursePage::InstructionsCard @title="How to pass this stage" id="second-stage-instructions-card" ...attributes>
+<CoursePage::InstructionsCard
+  @title="How to pass this stage"
+  id="second-stage-instructions-card"
+  ...attributes
+  {{did-update this.loadLanguageGuides @courseStage}}
```

Comment:
@andy1li this means that two components will be individually trying to load language guides on the same page. Can we move this into the `instructions.ts` controller please? Once it's loaded there, other components can just use methods that do something similar to what we do for `solutions`: 

<img width="852" alt="Screenshot 2024-10-29 at 11 31 32" src="https://github.com/user-attachments/assets/45c1b7af-c4e1-4ceb-b28a-a3f8eec3cc97">